js/widget.ts

```
@@ -193,25 +185,21 @@ async function render({ model, el }: RenderProps) {
-    }
-
-    await cosmograph.setConfig(cosmographConfig)
-    const pointColorType = getPointColorLegendType(stats.pointsSummary, cosmographConfig)
-    await legends.updateLegend('point', 'color', pointColorType)
-    await legends.updateLegend('link', 'color')
-
+    legends.update('point', 'size')
+    legends.update('point', 'color')
+    legends.update('link', 'width')
+    legends.update('link', 'color')
+    console.log(`onGraphRebuilt model.get('point_timeline_by')`, model.get('point_timeline_by'))
```

ESLint

[error] Unexpected console statement. Only these console methods are allowed: warn, error.

(no-console)